### PR TITLE
Action menu react

### PR DESCRIPTION
**Pull request by ptbrowne**
Tweaks to make the action menu work with React.

### CODE REVIEW

File: react/ActionMenu/index.jsx [hunk 1]
```diff
@@ -125,13 +129,11 @@ class ActionMenu extends Component {
...
   handleMenuRef = menu => {
-    // FIXME
-    this.menuNode = ReactDOM.findDOMNode(menu) // eslint-disable-line react/no-find-dom-node
+    this.menuNode = menu
   }
 
   handleWrapperRef = wrapper => {
-    // FIXME
-    this.wrapperNode = ReactDOM.findDOMNode(wrapper) // eslint-disable-line react/no-find-dom-node
+    this.wrapperNode = wrapper
```
Comment (CPatchane) [inline react/ActionMenu/index.jsx]: I think we can now put them directly in the render code

Comment (ptbrowne) [reply react/ActionMenu/index.jsx]: I prefer not to create functions in the render method. It's not based on scientific benchmarks.

Comment (CPatchane) [reply react/ActionMenu/index.jsx]: Ok

**Comment by ptbrowne** [comment]
Is it OK ?

**Pull request merged**